### DISCUSSION TITLE
Add support for subtrees

### DISCUSSION THREAD

**Discussion by eng-marani** [gitextensions/gitextensions#11588] [Ideas]
Subtree(s) are not first citizens in gitextensions.
The request is that the tool should have full support for subtree as a variant of modules, seamlessy integrating them into the user interface. Actually, these are no cue that subtree even exists, and the only way to create/remove them is by using git command line.

**Comment by pmiossec** [comment]
Duplicate of #1959

For such feature, the one that use the feature is the best placed to develop it.
Personally, I never used it and I tried to read about it 
So PR is welcome.

If you really can't do a PR, a good idea is to describe in as much details as possible how you see the feature integrated in GE.
And even better with the git commands that should be handled.

**Reply by eng-marani** [reply]
https://www.atlassian.com/git/tutorials/git-subtree
